framework/moose.mk

```diff
@@ -12,6 +12,20 @@ ifneq (,$(findstring darwin,$(libmesh_HOST)))
 	libmesh_LDFLAGS += -headerpad_max_install_names
 endif
 
+#libmesh conda check
```

Comment:
I put them up here because I assumed we wanted them before the check for Conda verification which directly follows them.

Reply:
That conda verification is currently skipped by @milljm, as it was intended for conda package maintainers early on in our package creation cycle. It's been left behind so that we might use it again later.

Regardless, I see your logic - don't mind you leaving it here.